community/libmupdf/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < pkcs7-fix.patch
```

Comment:
What does the patch do?

Reply:
I believe it's a libressl fix. Build fails without it.